source/statements.tex

```diff
@@ -759,25 +768,28 @@
 \indextext{statement!\idxcode{break}}
 
 \pnum
-The \keyword{break} statement shall occur only in an
+A \keyword{break} statement shall be enclosed by\iref{stmt.pre}
 \indextext{\idxgram{iteration-statement}}%
 \indextext{statement!\idxcode{switch}}%
-\grammarterm{iteration-statement} or a \keyword{switch} statement and causes
-termination of the smallest enclosing \grammarterm{iteration-statement} or
-\keyword{switch} statement; control passes to the statement following the
+an \grammarterm{iteration-statement}\iref{stmt.iter} or
+a \keyword{switch} statement\iref{stmt.switch}.
+The \keyword{break} statement causes
+% FIXME: What does it mean for one statement to be "smaller" than another?
```

Comment:
@burblebee, @jensmaurer: is this really unclear? "Encloses" forms a partial order, and "smallest" is with respect to that order. Do we need to spell that out as part of [stmt.pre]? Or just say "smallest <ins>relative to the partial ordering determined by "encloses"</ins>" here?

Reply:
I think this is sufficiently clear in context. We could replace "smallest" with "nearest", if that would help, but I'm hesitant to do more than that editorially.

Reply:
OK. Let's just leave it as is. @burblebee: Feel free to send a follow-up PR if you feel that this is really unclear.